apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/db/BrevRepository.kt

```diff
@@ -431,9 +431,10 @@ class BrevRepository(private val ds: DataSource) {
             INNER JOIN innhold i on b.id = i.brev_id
             WHERE b.id = ?
             AND h.id IN (
-                SELECT DISTINCT ON (brev_id) id
-                FROM hendelse
-                ORDER  BY brev_id, opprettet DESC
+                SELECT DISTINCT ON (h2.brev_id) h2.id
+                FROM hendelse h2
+                WHERE h2.brev_id = b.id
+                ORDER BY h2.brev_id, h2.opprettet DESC
```

Comment:
I stedet for DISTINCT ON kan vel LIMIT 1 benyttes. Usikker på hvorvidt det vil påvirke ytelsen merkbart.